### PR TITLE
Add health facilities view

### PR DESCRIPTION
**Pull request by Awinja-j**
**This Pull Request Introduces the health facilities Api View**

**Description of Task to be completed:**

The task required writing tests for health facilities and its end points.
**Manual testing:**

You can manually test it by running ```nosetests``` or ```python health_facilities.py``` in the test folder.

**Relevant Trello stories?**

[HTools API] Add Health Facilities view

### CODE REVIEW

**Comment by DavidLemayian** [comment]
@Awinja-Andela I'm pretty sure one can't test by running `python health_facilities.py.`

Please improve your description. I've also just updated the title to start the sentence with a capital letter.. There is a penchant to good descriptions and titles for PRs that translates to the same care in code.

File: healthtools_ke_api/views/health_facilities.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+app = Flask(__name__)
+
+app.register_blueprint(doctors_api, url_prefix='/doctors')
+app.register_blueprint(nurses_api, url_prefix='/nurses')
+app.register_blueprint(clinical_officers_api, url_prefix='/clinical-officers')
+app.register_blueprint(health_facilities_api, url_prefix='/health-facilities')
+
+app.register_blueprint(sms_handler)
```
Comment (DavidLemayian) [inline healthtools_ke_api/views/health_facilities.py]: Wait.. no.....

**Review by DavidLemayian — COMMENTED** [review]
😱

**Comment by DavidLemayian** [comment]
@Awinja-Andela I actually had to check the rest of the views and check if we really are writing them the same way. And no, no we are not.. Please review the other views first. (Except Telegram Bot. That is very experimental WIP.)

**Comment by DavidLemayian** [comment]
@Awinja-Andela Please fix merge conflicts.

**Comment by Awinja-j** [comment]
@DavidLemayian merge conflicts resolved

**Comment by Awinja-j** [comment]
@DavidLemayian Kindly check on this.

**Comment by DavidLemayian** [comment]
Hi @Awinja-Andela. Please add missing documentation for this.

**Comment by Awinja-j** [comment]
@DavidLemayian i have added a comment on the search function.

**Comment by DavidLemayian** [comment]
Hi @Awinja-Andela I don't think the `views/response.py` file is needed. It's what is in `build_query.py` already. Please pull latest changes and fix this.

**Comment by Awinja-j** [comment]
@DavidLemayian i have removed the file.

**Pull request closed without merging**